test/Microsoft.TestPlatform.AcceptanceTests/AcceptanceTestBase.cs

```diff
@@ -51,6 +52,17 @@ public class AcceptanceTestBase : IntegrationTestBase
     public const string NETCORE21_50 = "netcoreapp2.1;netcoreapp3.1;net5.0";
     public const string NETFX452_NET50 = "net452;net461;net472;net48;netcoreapp2.1;netcoreapp3.1;net5.0";
     public const string NETFX452_NET31 = "net452;net461;net472;net48;netcoreapp2.1;netcoreapp3.1";
+    public const string DEFAULT_RUNNER_NETFX = "net451";
+    /// <summary>
+    /// Our current defaults for .NET and .NET Framework.
+    /// </summary>
+    public const string DEFAULT_RUNNER_NETFX_AND_NET = $"{DEFAULT_RUNNER_NETFX};netcoreapp2.1";
```

Comment:
Why use string concatenation rather than string interpolation for the first one? 

I don't want to mix the old constants with the new ones. Because I want to remove those old data sources, and move everything that makes sense to IntegrationTestBase. I am not even entirely convinced that using the DEFAULT_RUNNER_NETFX constant was not a mistake here.

Reply:
If you plan to then remove the old constants then yeah please ignore this.

Regarding the concatenation instead of the string interpolation it's just that in this case (only 1 char) it felt less readable to see `$"{DEFAULT_RUNNER_NETFX};{Core21TargetFramework}"` than `DEFAULT_RUNNER_NETFX + ";" + Core21TargetFramework`.